integration-tests/cypress/e2e/project/edit/document-context-menu.cy.js

```diff
@@ -79,8 +79,8 @@ describe('/projects/:projectId/edit - Document Context Menu', () => {
     cy.getByTestId('robot-more').click();
     cy.getByTestId('treeitem-contextmenu').findByTestId('rename-tree-item').click();
 
-    cy.getByTestId('name-edit').type('renamed-robot{enter}');
-    cy.getByTestId('renamed-robot').should('exist');
+    cy.getByTestId('name-edit').type('.renamed{enter}');
+    cy.getByTestId('robot.renamed').should('exist');
```

Comment:
Why does the test needs to be changed? Did the behavior change? It should not but from what I understand here, it seems that when you start typing in the textfield, you are not replacing the original content but instead appending to the original content, is that the case?

Reply:
Yes, ExplorerInitialDirectEditTreeItemLabelProvider in sirius web package return the current label, should I return an empty string ?